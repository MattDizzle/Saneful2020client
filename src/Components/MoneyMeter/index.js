import React from 'react';

import './MoneyMeter.scss';

const MoneyMeter = (props) => {

  return (
    <p>{`Money: $${props.currentMoney}`}</p>
  );
};

export default MoneyMeter;