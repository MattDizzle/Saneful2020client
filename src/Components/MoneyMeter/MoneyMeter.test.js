import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import MoneyMeter from './index';


describe(`MoneyMeter Component`, () => {
    describe(`Smoke test`, () => {
        it(`Renders without crashing`, () => {
            const div = document.createElement('div');
            ReactDOM.render(
                <BrowserRouter>
                    <MoneyMeter />
                </BrowserRouter>
                , div);
            ReactDOM.unmountComponentAtNode(div);
        });
    });
    
    describe(`Snapshot test`, () => {
        it(`Renders the UI as expected`, () => {
            const tree = renderer
                .create(
                    <BrowserRouter>
                        <MoneyMeter />
                    </BrowserRouter>
                )
                .toJSON()
            expect(tree).toMatchSnapshot()
        });
    });
})