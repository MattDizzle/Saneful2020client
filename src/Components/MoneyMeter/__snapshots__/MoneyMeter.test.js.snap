// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoneyMeter Component Snapshot test Renders the UI as expected 1`] = `
<p>
  Money: $undefined
</p>
`;
